bin/check-all-gpg-keys-for-expiry.rb

```diff
@@ -0,0 +1,348 @@
+#! /usr/bin/env ruby
+#
+# check-all-gpg-keys-for-expiry
+#
+# DESCRIPTION:
+#   This script will check all keys' expiry dates and generate JSON message or output
+#   the string "CheckAllGpgKeysForExpiry OK: ..." (like a Nagios plugin). JSON message can be
+#   sent to Sensu client.
+#
+# OUTPUT:
+#   plain text
+#   Defaults: CRITICAL if a key is about to expire in 5 days
+#             WARNING if a key is about expire in 14 days
+#             GPG home directory is /root/.gnupg
+#             GPG executable binary is located under /usr/bin/gpg
+#             It checks secret keys
+#             It displays JSON message
+#
+# PLATFORMS:
+#   Linux, Mac OS X
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin, csv, open3, date, time, json
```

Comment:
of those only `sensu-plugin` is a gem, can you remove the rest that are in the stdlib.

Reply:
and `json`

Reply:
True tho there's no need to explicitly depend on `json` since it gets pulled in via `sensu-plugin`.

Reply:
I have removed `json`.